src/main.jsx

```diff
@@ -1,10 +1,13 @@
 import React from 'react'
 import ReactDOM from 'react-dom/client'
 import App from './App'
+import Calculator from './Components/Calculator'
+import Button from './Components/Button'
 import './index.css'
 
 ReactDOM.createRoot(document.getElementById('root')).render(
   <React.StrictMode>
-    <App />
+    <Calculator />
+    {/* <Button number={2} type={'operator'}/> */}
```

Comment:
- [ ] Kindly remove all unnecessary commented-out codes from your file. Doing so makes your code look clean and readable. 👍

Reply:
This change has been done well